Formula/mysql.rb

```diff
@@ -123,13 +123,27 @@ def post_install
       system bin/"mysqld", "--initialize-insecure", "--user=#{ENV["USER"]}",
         "--basedir=#{prefix}", "--datadir=#{datadir}", "--tmpdir=/tmp"
     end
+
+    # Create my.cnf that binds to 127.0.0.1 by default
+    unless (etc/"my.cnf").exist?
```

Comment:
Thanks for the feedback on this! I really appreciate you sticking with me through this :)

To use `(buildpath/"my.cnf").write` looks like this logic needs to be moved to the `install` function? I found [the git](https://github.com/Homebrew/homebrew-core/blob/master/Formula/git.rb#L158) [and uwsgi](https://github.com/Homebrew/homebrew-core/blob/master/Formula/uwsgi.rb#L64) formulas do a similar thing but they do this in the `install` function. I tried leaving the logic in the `post_install` function but while there are no crashes, the file doesn't get copied. Additionally, `buildpath` is null when doing anything other than `--build-from-source install` (e.g. `brew postinstall mysql` crashes because `buildpath` is null).

I've checked in the updated code using `buildpath` var and moved the logic to the `install` function. Let me know if this is the correct approach?